server/src/yml-objects/AbstractYmlObject.ts

```diff
@@ -162,14 +161,14 @@ function getType(fieldOptions: FieldContext[], connection: IConnection, baseType
     try {
         for (const element of fieldOptions.filter((elem) => !!elem.commonField)) {
             const option = element.commonField();
-            if (!option || !option._optionName || !option._optionValues || option._optionValues.length === 0) {
+            if (!option || !option._optionName || !option._optionValue) {
                 continue;
             }
             const optionName = option._optionName.text;
             if (optionName === 'domains') {
-                domains = option._optionValues[0].text;
+                domains = option._optionValue.text;
             } else if (optionName === 'domainsLevel2') {
-                domainsLevel2 = ` − ${option._optionValues[0].text}`;
+                domainsLevel2 = ` − ${option._optionValue.text}`;
```

Comment:
Wooo not sure how you did that, but glad to see some of those puzzling `[0]`-s bite the dust.

Reply:
Now, each object option (the arrow thingy) can have only one value. In the previous version, it could be multiple values due to a weird and now fixed rule in the grammar.